src/components/carousel/index.jsx

```diff
@@ -195,6 +198,11 @@ const Carousel = ({
 			{...handlers}
 		>
 			<div className={`${COMPONENT_CLASS_NAME}__top-actions`}>
```

Comment:
Calling this div `*__top-actions` is inappropriate imo.  User may want to move this to the bottom or to the right/left and vertically oriented.  This class needs a more generic name that doesn't suggest placement.

Reply:
that's a good point. will adjust the naming. maybe controls?

Reply:
@visionbegin updated -- thanks!

Reply:
LGTM!